lib/vscode/src/typings/native-keymap.d.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
If we don't need it, how come we are adding in the types? Do we need this for some reason?

Reply:
It's kind of like what we're doing for electron - we need the types for it to compile correctly - but then it never gets loaded, so we don't need the module itself - _just_ the types so tsc won't complain.